rickroll.py

```diff
@@ -153,5 +174,55 @@ def original():
 
     return str(play_tune(tune))
 
+@app.route("/sms", methods=["POST"])
+def sms():
+    """Adds an incoming SMS to a queue, to reply to later."""
+    bucket.put_object(
+        Key='queue/{to}/{from_}'.format(to=request.form['To'],
+                                        from_=request.form['From']),
+        Body='',
```

Comment:
Yes, that's exactly how it works! (I meant to add a PR comment summarising how everything works, but ran out of time.)